target/linux/sunxi/image/cortex-a7.mk

```diff
@@ -31,6 +31,16 @@ define Device/friendlyarm_nanopi-m1-plus
 endef
 TARGET_DEVICES += friendlyarm_nanopi-m1-plus
 
+define Device/sun8i-h3-nanopi-r1
+  DEVICE_TITLE:=FriendlyArm NanoPi R1
+  DEVICE_PACKAGES:=kmod-rtc-sunxi kmod-usb2 kmod-usb-net kmod-usb-net-rtl8152 \
+	kmod-leds-gpio kmod-ledtrig-heartbeat \
```

Comment:
Please fill up lines until 80 chars, there is room for at least one additional package in this line.